crates/mdman/src/main.rs

```diff
@@ -98,15 +98,16 @@ fn process_args() -> Result<Options, Error> {
                 let man = args
                     .next()
                     .ok_or_else(|| format_err!("--man requires a value"))?;
-                let parts: Vec<_> = man.splitn(2, '=').collect();
-                let key_parts: Vec<_> = parts[0].splitn(2, ':').collect();
-                if parts.len() != 2 || key_parts.len() != 2 {
-                    bail!("--man expected value with form name:1=link");
-                }
```

Comment:
The old code errored on having too many `=` or `:` which the new code no longer does. Do we care?

Reply:
I don't fully understand the format but we can keep the behavior the same for this refactor